lua/toggleterm.lua

```diff
@@ -60,14 +60,14 @@ end
 --Create a new terminal or close beginning from the last opened
 ---@param _ number
 ---@param size number
----@param directory string
+---@param dir string
 ---@param direction string
-local function smart_toggle(_, size, directory, direction)
+local function smart_toggle(_, size, dir, direction)
```

Comment:
This is a fairly minor point but why rename this to `dir` rather than directory, tbh the only reason I've used dir anywhere was to make things a little more ergonomic so users don't have to type `directory` all the time. I don't do the same with direction because of the clash and because frankly the direction flag was a bit of an afterthought

Reply:
I guess maybe it's for consistency in which case that's fine